src/common/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const Button = ({ type, className, onClick, isDisable, children }) => {
+  return (
+    <button
+      type={type}
+      className={className}
```

Comment:
Rather than passing `classname` as prop for the common component, declare all the styling for the component here and use `variant` prop to accommodate different `styles` for the component.